.claude/skills/jira-evaluate-blockers/SKILL.md

```
@@ -0,0 +1,577 @@
+## Mode B: Evaluate
+
+Find issues with existing needs-\* labels or Blocked=True, and check whether the blocking condition has been resolved.
+
+### Discovery JQL Patterns
+
+All values come from `jira-project-reference.md`. Include `resolution = Unresolved` by default.
+
+All queries include the standard filters from `jira-project-reference.md` § Standard Query Filters (type, epic link, team).
+
+**Issues with needs-\* labels:**
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)